hms-catalog/src/test/resources/auth/jwt/jwt-authorized-key.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why does this key file belong to the repository? It seems like sensitive keys. Or is this needed for the testing?

Reply:
It is a resource for tests when we mock JWT flow.